test/askContainer/treeViews/treeViewProviders/helpViewProvider.test.ts

```diff
@@ -0,0 +1,157 @@
+import * as vscode from "vscode";
+import * as assert from "assert";
+import * as sinon from "sinon";
+
+import { HelpViewProvider } from "../../../../src/askContainer/treeViews/treeViewProviders/helpViewProvider";
+import { HelpView } from "../../../../src/askContainer/treeViews/helpView";
+import { ext } from "../../../../src/extensionGlobals";
+import { PluginTreeItem, CustomResource, ContextValueTypes, Resource } from '../../../../src/runtime/';
+import { HELP_VIEW_ITEMS, EXTERNAL_LINKS } from '../../../../src/constants';
+import { Logger } from '../../../../src/logger';
+
+describe("TreeView_helpViewProvider tests", () => {
+    let helpView: HelpView;
+    let helpViewProvider: HelpViewProvider;
+    let sandbox: sinon.SinonSandbox;
+    let debugSpy: sinon.SinonStub;
+    let verboseSpy: sinon.SinonStub;
+    before(() => {
+        helpView = new HelpView(ext.context);
+        helpViewProvider = new HelpViewProvider(helpView);
+    });
+
+    beforeEach(() => {
+        sandbox = sinon.createSandbox();
+        debugSpy = sandbox.stub(Logger, 'debug');
+        verboseSpy = sandbox.stub(Logger, 'verbose');
+    });
+
+    afterEach(() => {
+        sandbox.restore();
+    });
+
+    describe("getChildren", () => {
+        it ('when no element', () => {
```

Comment:
nit: The name of test cases can be improved :)

Reply:
Good call, will better improve test names